modules/mqtt/mqtt-parser.c

```diff
@@ -36,6 +36,7 @@ static CfgLexerKeyword mqtt_destination_keywords[] =
   { "topic", KW_TOPIC },
   { "keepalive", KW_KEEPALIVE },
   { "qos", KW_QOS },
+  { "message_template", KW_MESSAGE_TEMPLATE },
```

Comment:
I think in case of destinations, we usually just call this `template`.

Reply:
fixed by this commit: 2a8fe691bf49900e9a2b5e8e2a7297573f1b70b4